pyfstat/helper_functions.py

```diff
@@ -429,6 +429,74 @@ def convert_array_to_gsl_matrix(array):
     return gsl_matrix
 
 
+def get_sft_as_numpy_array(sftfilepattern, fMin=None, fMax=None, constraints=None):
+    """
+
+    Parameters
+    ----------
+    sftfilepattern: str
+            Pattern to match SFTs using wildcards (`*?`) and ranges [0-9];
+            multiple patterns can be given separated by colons.
+    fMin, fMax: float or None
+        Restrict frequency range to `[fMin, fMax]`.
+        If None, retreive the full frequency range.
+
+    Returns
+    ----------
+    times: np.ndarray
```

Comment:
Order does not match actual returns, and is the actual one different on purpose from the old func?

Reply:
Time and frequency are interchanged to return  with the following signature: `1D, {1D per ifo}, {2D per ifo}`.
I think it makes more sense to split, between container type, but can revert if you feel otherwise.